eb_hooks.py

```diff
@@ -136,6 +136,8 @@ def post_ready_hook(self, *args, **kwargs):
         parallel_param = 'max_parallel'
```

Comment:
A better with EasyBuild 5.x would be to use `self.cfg.parallel` instead of `self.cfg['parallel']`

Reply:
In general, I think the information gathering here is in the wrong place. You only need to do this if you plan to change the parallelism, so this whole block could be moved to when you need it, right before `if new_parallel != parallel:`